docs/user/pings/index.md

```diff
@@ -104,6 +104,34 @@ A pre-defined set of headers is additionally sent along with the submitted ping:
 ## Defining background state
 
 These docs refer to application 'background' state in several places.
-This specifically means when the activity is no longer visible to the user, it has entered the Stopped state, and the system invokes the [`onStop()`](https://developer.android.com/reference/android/app/Activity.html#onStop()) callback.
-This may occur, for example, when a newly launched activity covers the entire screen.
+
+{{#include ../../tab_header.md}}
+
+<div data-lang="Android" class="tab">
```

Comment:
It's sort of not-quite-the-right-thing, but on other pages we've used `Kotlin` and `Swift` rather than `Android` and `iOS`.  The reason it matters is that the user's preference is stored in local storage and follows them from page to page.

As part of the larger discussion about we better talk about languages vs. platforms, we could address that then...

Reply:
Yeah, it felt a little _too_ specific to use the languages, but the preference trumps the tab title, I agree.